### DISCUSSION TITLE
Changing background color of tooltips

### DISCUSSION THREAD

**Discussion by mgriff88** [squidfunk/mkdocs-material#7519] [Q&A] [answered]
Hi all, I'm a bit stumped on changing the background color of tooltips, specifically `md-tooltips2`.

It's straightforward in `md-tooltips` and works no problem. But in tooltip2, it creates this big line across the entire page. When I look at the source, it's like there is no background-color variable? But none of the other background-color options work either. https://github.com/squidfunk/mkdocs-material/blob/master/src/templates/assets/stylesheets/main/components/_tooltip2.scss

<img width="320" alt="image" src="https://github.com/user-attachments/assets/b84d475a-c916-40dc-b2e0-1c597740f0ea">

In my example below, I used red as the background just to make it very clear what's happening.

<img width="473" alt="image" src="https://github.com/user-attachments/assets/aea8cf1b-5ba6-4871-9ac6-561ee4610291">

Anyone know what's needed here? Thanks in advance.

**Comment by squidfunk** [comment, marked as answer]
This should do the trick:

``` css
.md-tooltip2__inner {
  background: red;
}
```

**Reply by mgriff88** [reply]
Bingo, that worked! Thanks for the quick reply, @squidfunk !
